### DISCUSSION TITLE
Linux: "BASS faulted with error code 3: Driver." error on start up, fresh install.

### DISCUSSION THREAD

**Discussion by aimaismog** [ppy/osu#29034] [Q&A] [answered]
### Type

Game behaviour

### Bug description

I installed osu!lazer literally today, and it just showed this error and no audio on startup.
I have Pipewire installed, unless PulseAudio is required for some reason?
Pipewire is confirmed to be on latest version (1.2.1-1)

### Screenshots or videos

![image](https://github.com/user-attachments/assets/9e7dfce6-4eaf-487a-9842-b5cda647276c)

### Version

Latest

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/16356262/compressed-logs.zip)

**Comment by Stoppedpuma** [comment, marked as answer]
Make sure you have pipewire-alsa installed.

**Reply by aimaismog** [reply]
i see... this should've been installed when i installed pipewire right? weird.

**Reply by Stoppedpuma** [reply]
Going off the fact that the arch package has "(optional)" next to it maybe not?

**Reply by aimaismog** [reply]
yep wasn't installed for some reason

**Reply by uwuclxdy** [reply]
thanks :sob: :pray:
